cms/static/js/views/course_info_update.js

```diff
@@ -73,6 +73,13 @@ define(["js/views/baseview", "codemirror", "js/models/course_update",
 
         onSave: function(event) {
             event.preventDefault();
+            validation = viewUtils.keywordValidator.validate_string(this.$codeMirror.getValue());
+            if (!validation.is_valid) {
+                var message = gettext("There are invalid keywords in your message. Please check the following keywords and try again:");
+                message += "\n" + validation.invalid_keywords;
```

Comment:
As in the feedback for the email, can we have the invalid keywords appear on separate lines? Please look at how this was resolved in the last PR.

How it looks now:
![screen shot 2015-01-06 at 3 14 42 pm](https://cloud.githubusercontent.com/assets/1985317/5635395/dafb17bc-95b6-11e4-9539-9947100224ec.png)